homeassistant/components/rest/binary_sensor.py

```diff
@@ -59,6 +64,13 @@ async def async_setup_platform(
 
     if rest.data is None:
         if rest.last_exception:
+            if isinstance(rest.last_exception, ssl.SSLError):
+                _LOGGER.error(
+                    "Error connecting %s failed with %s",
+                    conf[CONF_RESOURCE],
+                    rest.last_exception,
+                )
+                return
```

Comment:
Is this going to log over and over and as it retires?

Reply:
I thought we want to fail the setup in case of SSL issues? Ther for it just returns, but do not raise `PlatformNotReady`

Reply:
Sorry my bad. I lost context when flipping between scenarios